static/app/components/discover/performanceCardTable.tsx

```diff
@@ -677,14 +677,14 @@ function PerformanceCardTable({
   );
 }
 
-type Props = AsyncComponent['props'] & {
+interface Props extends AsyncComponentProps {
```

Comment:
Is using `AsyncComponentProps` faster? I would actually expect `AsyncComponent['props']` to really be no different

Reply:
No, but we have a lint rule that converts `interface Props extends AsyncComponent["props"]` to `interface Props extends AsyncComponent.props` which is not functionally equivalent in TS land and we end up with this
<img width="697" alt="CleanShot 2022-05-06 at 15 04 24@2x" src="https://user-images.githubusercontent.com/9317857/167201810-a6f8783b-a26c-4ba0-9bd7-be60c922aa2e.png">

Reply:
oh shoot, that's broken!